packages/xo-web/src/common/editable/index.js

```diff
@@ -341,74 +332,80 @@ export class Number extends Component {
   }
 }
 
-export class Select extends Editable {
+class SimpleSelect_ extends Editable {
   static propTypes = {
-    options: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
-      .isRequired,
     renderer: PropTypes.func,
```

Comment:
Let's call it `optionRenderer` otherwise we'd need to forward it explicitly to `FormSelect`.